bundle.yaml

```diff
@@ -469,39 +475,39 @@ spec:
         - name: OPERATOR_NAME
           value: sumologic-kubernetes-collection-helm-operator
         - name: RELATED_IMAGE_KUBERNETES_SETUP
-          value: registry.connect.redhat.com/sumologic/kubernetes-setup@sha256:ca76893ae48cfc0e6ee4ef24ccbeb4a90aad738308feacdbd9eed205826406b9
+          value: public.ecr.aws/sumologic/kubernetes-setup@sha256:767fd3505df5ca59fb502dfb1678f3d6e8cfac2d249972cc50d541939a7a2e31
```

Comment:
Are you temporarily switching these?

Reply:
Yes, at this moment new images are not available in `registry.connect.redhat.com` so to test with new images I need to switch temporarily to public container registries.